ssl_league_msgs/msg/VisionDetectionRobot.msg

```diff
@@ -0,0 +1,6 @@
+float32 confidence
+uint32 robot_id
+geometry_msgs/Point32 pos
```

Comment:
Thoughts on combining `pos` and `orientation` using `geometry_msgs/msg/Pose2D`? 
https://docs.ros2.org/galactic/api/geometry_msgs/msg/Pose2D.html

Reply:
Pose2D looks to be depreceated in Foxy, but I couldn't figure out a one liner to do yaw->quat so I'm cool just doing Pose2D until it finally dies and we swap.

Reply:
Good catch, I completely missed that. 
One way to do yaw->quat is to use the utilities in `tf2_geometry_msgs`:
```
auto quat_msg = tf2::toMsg(tf2::Quaternion(tf2::Vector3(0,0,1), yaw));
```